### PR TITLE
[System.Configuration] SectionInformation.GetRawXml() returns null

### LINKED ISSUE #13599: [System.Configuration] SectionInformation.GetRawXml() returns null
The method `SectionInformation.GetRawXml()` returns `null` even if the section is really present in config-file.

### PR DESCRIPTION
**Pull request by NVoronchev**
Fixed #13599

### CODE REVIEW

File: mcs/class/System.Configuration/System.Configuration/SectionInformation.cs [hunk 1]
```diff
@@ -201,12 +203,6 @@ public void UnprotectSection ()
 			protection_provider = null;
 		}
 
-		public void SetRawXml (string rawXml)
```
Comment (filipnavara) [inline mcs/class/System.Configuration/System.Configuration/SectionInformation.cs]: This is public API. You cannot just remove it.

Comment (NVoronchev) [reply mcs/class/System.Configuration/System.Configuration/SectionInformation.cs]: My fault. Apologize.

**Comment by NVoronchev** [comment]
@filipnavara, @marek-safar, Please, find updates of the current review.

**Comment by marek-safar** [comment]
approve

**Comment by NVoronchev** [comment]
Dear colleagues, should I do something in this request or we wait for additional review?

**Comment by marek-safar** [comment]
@NVoronchev it fails to build due to
```
System.Web.Configuration_2.0/CustomErrorsSection.cs(68,36): error CS0115: 'CustomErrorsSection.DeserializeSection(XmlReader)': no suitable method found to override
```

**Comment by steveisok** [comment]
When https://github.com/mono/mono/pull/12619 lands, System.Configuration will be corefx based.  Based on that, we're going to have to close this PR.  

Thanks for your contribution 👍

**Pull request closed without merging**